torch/csrc/jit/import.cpp

```diff
@@ -402,15 +402,15 @@ ModuleDecoder::ModuleDecoder(
 void import_ir_module(
     ModuleLookup module_lookup,
     std::istream& in) {
-  ModuleDecoder(module_lookup, in);
+  ModuleDecoder(module_lookup, in); // NOLINT(bugprone-unused-raii)
```

Comment:
Would giving a name to this temporary fix the lint? That seems simpler than the `NOLINT` comment

Reply:
ok